app-text/kbbi-qt/kbbi_qt-1.1.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="An Indonesian Dictionary KBBI based on QT"
+DESCRIPTION="Indonesian dictinary version of Kamus besar bahasa Indonesia"
+COPYRIGHT="
+2017-2020 Anto Samalona
+2017-2020 Ade Malsasa Akbar
+2017-2020 Azis Pratama
+2017-2020 Billy
+2017-2020 Christian Kurniawan
+2017-2020 HanHan
+2017-2020 La Ode Muh. Fadlun Akbar
+2017-2020 Ramdziana Feri Yustianto
+2017-2020 Rania el-Amina
+2017-2020 Sucipto
+"
```

Comment:
Move line 4 up to line 3, add tabs on the next lines to indent

Reply:
ok done